### ISSUE TITLE
Notes from a quick read through

### ISSUE THREAD

**Issue by pilegard** [langcog/experimentology#114]
Thank you all for this excellent book! I teach an undergraduate research methods lab and am thinking of adopting portions of it there. So my big comment is this is great and much appreciated. My small comment is that I just finished a quick read-through and took notes on things I thought might warrant a look (some errors, some impressions). Also I'm not well-versed in GitHub etiquette so I hope this is a good way to send feedback:

- [x] - Chapter 1 footnote 4: absence misspelled “absenece”
- [ ] - Chapter 2, text reference to Figure 2.3 says “notice that the arrow connecting distance and generalization is doing a lot of work”, but node on the figure is labeled “similarity” not “distance”
- [x] - Section 2.4, something is missing (word(s) after “hypothesis”?) or this sentence is hard to parse: “Instead, the reasoning about whether a hypothesis relies on a large series of what are called auxiliary hypotheses that are necessary to link the observation to the theory (Lakatos, 1976).”
- [x] - Section 2.4, meant to say “may not” rather than “may”?: “An unexpected observation may cause us to give up on a main hypothesis in our theory – but it will often cause us to question our auxiliary assumptions instead”
- [x] - Chapter 3, “Depth” box: “Errington et al. (2021)” should be parenthetical citation
- [x] - Section 3.3 “Causes of replication failure” should maybe be one heading level lower (i.e., a subsection of section 3.2, “Replication”)
- [x] - Chapter 3, “Accident report” box: think the first Simmons 2018 citation is supposed to point to Simmons 2011 instead **[TH NOTE - I think Simmons 2018 is correct - its a paper where they discuss the 2011 paper]**
- [x] - Section 3.4.3 (“Open Science”) broken figure reference (“Figure @(fig:replication-umbrella”)
- [x] - Section 4.1.2 (“Deontological approaches”), second paragraph: “IRB” acronym used first time without explanation
**[TH NOTE - also I think IRB is predominantly a US term, so perhaps we just switch to saying "university ethics board"?]**
- [x] - Chapter 4, “Tuskegee Syphilis Study” box: “accident report” may be a bit cute for the box style
- [x] - Chapter 4: The term “confederate” isn’t defined in text and it is notably confusing to google
**[TH NOTE - I've just changed it to 'actor']**
- [x] - Section 4.3 paragraph starting with “As we’ll discuss”: broken reference (“see @ref(writing for details”)
- [x] - Chapter 5, paragraph starting with “For example” under the “Maximum likelihood estimation” subheading: I think about half of those theta-hats should not have hats
- [x] - Chapter 5, paragraph starting with “The first part” under the “Bayesian estimation” subheading: I’m confused about some words here and how they relate to the example – it should refer to estimating tea quality but sometimes seems to refer to estimating the difference between treatments. Some elaboration would be helpful too (e.g., relationship b/w likelihood in Bayesian sense and maximum likelihood distribution – they’re equated in figures and would be useful to explain in text)
- [x] - Chapter 5, subheading “Estimating the treatment effect”: first paragraph never plainly equates treatment effect with beta; second paragraph doesn’t equate sigma with standard deviation until second use
- [x] - Figure 6.9: errant “null.” in caption
- [x] - Chapter 7 4th paragraph extra word: “datapoints are generated **via with** the flip of a weighted coin”
- [x] - Chapter 7 paragraph starting with “The term $\epsilon_i$ is the error term” second sentence wording (“**it not to** ‘error’ due to bias”) 
- [x] - Chapter 8 Internal validity: In my experience this term is more often used in the sense of the margin note, which is also the way it’s used in many intro research methods textbooks (e.g., Morling), and also the way it’s used in the Design chapter – I guess I think the marginal note should (1) point to the use of the term in the Design chapter and (2) shift emphasis (“often” rather than “sometimes”)
- [x] - Chapter 8: I expected to see something about inter-rater reliability here, either in the reliability section or in the “simple vs. complex behaviors” section (since it’s more important to assess with complex behaviors); it’s gestured toward in the survey measures depth box
- [x] - Chapter 8 Donders cited twice with two different years
- [x] - Chapter 9, sidenote 2: not obvious to me why IV/DV terminology is “transparently terrible”
- [x] - Chapter 9: internal validity is not clearly defined
- [x] - Chapter 9: nuisance factor not defined
- [x] - Section 14.1 Heading error (### The structure of a scientific paper)
- [x] - Appendix E section E.4.2.2, note left in: “(Mike – would this be okay? Feel free to adjust or delete if not).”
- [x] - General note: External links displayed as URLs often don’t work (e.g., chapter 7 Discussion Questions/Readings boxes)

**Comment by mcfrank** [comment]
Oh wow! Thank you so much for all this and for the kind comments. We are going to do a full revision this spring and your comments and fixes will be extremely (!!) helpful. Thanks again!!!

**Comment by TomHardwicke** [comment]
converted these suggestions to a checklist and ticking off if I've resolved

**Comment by TomHardwicke** [comment]
@mcfrank I've addressed all the one's I'm confident addressing and ticked them off

**Comment by mcfrank** [comment]
these were so helpful! thanks @pilegard
